OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -27,6 +27,11 @@ function abort() {
 :     "${OGG_HOME:?}"
 [[ -d "${OGG_HOME}"            ]] || abort "Deployment runtime, '${OGG_HOME}'. not found."
 
+:     "${SETUP_USER_SCRIPTS=${OGG_HOME}/scripts/setup}"
```

Comment:
Can these two variables be reduced to simply `OGG_DEPLOYMENT_SCRIPTS`?

Reply:
Combined into a single variable `OGG_DEPLOYMENT_SCRIPTS`